Source/Initialization/InjectorMomentum.H

```diff
@@ -47,6 +47,49 @@ private:
     amrex::Real m_ux_th, m_uy_th, m_uz_th;
 };
 
+// struct whose getMomentum returns momentum for 1 particle with relativistc drift velocity beta,
+// from the Maxwell-Juttner distribution.
+struct InjectorMomentumJuttner
+{
+    InjectorMomentumJuttner(amrex::Real theta, amrex::Real beta, int dir) noexcept
+        : t(theta), b(beta), d(dir)
+        {}
+
+    AMREX_GPU_HOST_DEVICE
+    amrex::XDim3
+    getMomentum (amrex::Real x, amrex::Real y, amrex::Real z) const noexcept
+    {
+        amrex::Real x1, x2, g;
+        amrex::Real u [3];
+        x1 = 0.;
+        g = 0.;
+        u[d] = 0.;
+        while(u[d]-g <= x1)
+        {
+            u[d] = -t*std::log(amrex::Random()*amrex::Random()*amrex::Random());
+            g = std::sqrt(1+std::pow(u[d],2));
+            x1 = t*std::log(amrex::Random());
+        }
+        x1 = amrex::Random();
+        x2 = amrex::Random();
+        u[(d+1)%3] = 2*u[d]*std::sqrt(x1*(1-x1))*std::sin(2*M_PI*x2);
+        u[(d+2)%3] = 2*u[d]*std::sqrt(x1*(1-x1))*std::cos(2*M_PI*x2);
+        u[d] = u[d]*(2*x1-1);
+        x1 = amrex::Random();
```

Comment:
@cameronjy Could you please add some comments here stating that this part does the volume-transformation and that we use the flipping method taking advantage of symmetry.

Reply:
I agree with @RevathiJambunathan. Also: could you add the reference of the paper that describes this, in the comment.

Reply:
@RemiLehe @RevathiJambunathan I made the changes -- let me know if the comments make sense or if they are too lengthy. I can elaborate details of my comments with respect to the Zenitani paper in person as well.